elements/rh-footer/test/rh-footer.spec.ts

```diff
@@ -142,7 +142,7 @@ const UNIVERSAL_FOOTER_TEMPLATE = html`
       <a href="#">*We’ve updated our privacy statement effective December 30, 202X.</a>
     </div>
   </rh-footer-universal>
-  <link rel="stylesheet" href="/elements/rh-footer/rh-footer-lightdom.css">
+  <link rel="stylesheet" href="/elements/rh-footer/rh-footer-lightdom.css" />
```

Comment:
please revert. [trailing slash is not necessary in HTML void elements](https://developer.mozilla.org/en-US/docs/Glossary/Void_element#self-closing_tags)